tests/validation/tests/v3_api/resource/terraform/longliving/aws/v2apps/main.tf

```diff
@@ -0,0 +1,137 @@
+terraform {
+    required_providers {
+        rancher2 = {
+            source  = "rancher/rancher2"
+            version = "1.10.6"
```

Comment:
We may want to update the rancher2 provider version since the latest version is 1.17.2 and it seems there are a number of bug fixes.

Reply:
Track the` .terraform.lock.hcl` file instead of hardcoding the rancher version here.